### ISSUE TITLE
Asteroids not spawning

### ISSUE THREAD

**Issue by smaier-at** [Kopernicus/Kopernicus#511]
Asteroid spawning handled by kopernicus is not working. (and comet spawning as well as mentioned in #503 )
workaround is to switch to the stock spawning system via change in Kopernicus_Config.cfg:
useKopernicusAsteroidGenerator from "true" to "Stock"

a bit more detail in this thread:
https://forum.kerbalspaceprogram.com/index.php?/topic/173466-sentinel-asteroid-mapping-stopped-working-solved/

**Comment by R-T-B** [comment]
I just saw that very same thread!  And it's true, I don't know how I missed it this long.  Anyhow, it should be fixed in release-72.  Would you mind testing?

**Comment by smaier-at** [comment]
My answer on the thread needs to be approved by a moderator, I just thought it would be good to open the github issue as well.

sure, I can test it tomorrow - is it also working for comets now? or just asteroids? (I really love the view of the comets - look awesome and I really want to visit one soon™)

**Comment by R-T-B** [comment]
Right now, just asteroids.  I will revisit comets with the customizable generator at a later date.  It was pretty tough to implement last time I tried, but I still plan on doing it, someday.

**Comment by smaier-at** [comment]
since you were that quick with the release and it's already on ckan I gave it a quick test with a new sandbox save and useKopernicusAsteroidGenerator set to true

Asteroids seem to spawn about every 100 days around kerbin (but I think they did the same with the last version as well as I previously captured a single one before figuring out the problem described above)

Having an active sentinel in kerbol orbit still leads to many exceptions like this, about every 2-5 ingame days:
`[EXC 00:50:32.924] NullReferenceException: Object reference not set to an instance of an object
	SentinelMission.SentinelScenario+<ProcessSentinelScan>d__15.MoveNext () (at <39c0323fb6b449a4aaf3465c00ed3c8d>:0)
	UnityEngine.SetupCoroutine.InvokeMoveNext (System.Collections.IEnumerator enumerator, System.IntPtr returnValueAddress) (at <12e76cd50cc64cf19e759e981cb725af>:0)
	UnityEngine.DebugLogHandler:LogException(Exception, Object)
	ModuleManager.UnityLogHandle.InterceptLogHandler:LogException(Exception, Object)
	UnityEngine.Debug:CallOverridenDebugHandler(Exception, Object)`
	
I think I saw very similar exceptions before, but didn't know they were related to kopernicus so I couldn't report them...

Anyway, I really need some sleep, so I hope I didn't mess something up with my versions during the test, but I can continue testing more into detail tomorrow or on the weekend :)

**Comment by smaier-at** [comment]
PS: my comment on the forum thread got approved as well, maybe it helps a bit...

**Comment by R-T-B** [comment]
I will leave this open to track the Sentinel exceptions, but for now, this is otherwise the intended behavior as of the moment.  Will need to work on those exceptions next release though, thanks.

**Comment by smaier-at** [comment]
I did another short test and ran the game with the stock asteroid spawner for a qualitative (and subjective) comparison with my test yesterday.

The most obvious difference is the spawning frequency. In stock, asteroids spawn about every other day around kerbin, so there are always around 10 in close proximity. With the kopernicus spawner the frequency is about one in 100 days (as stated in my previous comment).

And as I also stated, I got a single asteroid spawning some weeks ago with the previous version of kopernicus. So in my opinion, this "bug" wasn't a bug at all, just the very low spawning frequency made the impression that the system wasn't working at all. And it finally became a noticeable problem for me when I launched the Sentinels to scan for asteroids to do some cool missions but none were appearing...

On a side note, I think I saw a class H with the kopernicus spawner near kerbin, whereas the stock spawner only spawns up to class E in this region, and only comets in classes F to I. But that's fine for me :)
